services/garage-backend/src/routers/thingRouter.ts

```diff
@@ -1,21 +1,55 @@
 import express from 'express';
+import multer from 'multer';
+import * as fs from 'fs';
 
-import { requireAuth } from '../middlewares/auth';
 import {
   createThing,
   findThingById,
   updateThing,
   findAllThings,
   deleteThing,
+  restoreThing,
 } from '../controllers/thingController';
+import { requireAuth } from '../middlewares/auth';
 
 const router = express.Router();
 router.use(requireAuth());
 
+const storage = multer.diskStorage({
+  destination(req, file, cb) {
+    const dir = './uploads/';
+    fs.exists(dir, (exist) => {
+      if (!exist) {
+        return fs.mkdir(dir, (error) => cb(error, dir));
+      }
+      return cb(null, dir);
+    });
+  },
+
+  filename(req: any, file: any, cb: any) {
+    const filename = file.originalname;
+    const fileExtension = filename.split('.')[1];
+    cb(null, `${Date.now()}.${fileExtension}`);
```

Comment:
hmm, so the filename is unique by current date, could work. 
I suggest using `randomUUID` from `crypto` instead to be 101% safe
```suggestion
    import { randomUUID } from 'crypto'; // At the top of the file
    
    cb(null, `${randomUUID()}.${fileExtension}`);
```